rfc6962-bis.xml

```diff
@@ -1359,7 +1359,14 @@ but it is expected there will be a variety.
       </section>
       <section title="Auditing">
         <t>
-          Auditing is taking partial information about a log as input and verifying that this information is consistent with other partial information held. All clients described above may perform auditing as an additional function. The action taken by the client if audit fails is not specified, but note that in general if audit fails, the client is in possession of signed proof of the log's misbehavior.
+          Auditing ensures that the current published state of a log is reachable from previously published
+          states that are known to be good.  Audits are performed by monitors or TLS clients.
```

Comment:
Also, this is not really an accurate statement (e.g. it does not cover the inclusion proof case).

Maybe instead say "Auditing ensures that the information held by a client is consistent with the current published state of the log"?